documentation/modules/ROOT/pages/ztp-at-scale.adoc

```diff
@@ -40,9 +40,9 @@ MCE provides most of the functionality to provision OpenShift clusters, since it
 [#siteconfig]
 === SiteConfig Operator
 
-The SiteConfig Operator introduces a unified ClusterInstance API, which is derived from the SiteConfig API from the SiteConfig Generator kustomize plugin. This API decouples parameters that define a cluster from the manner in which the cluster is deployed. This separation removes certain limitations that are presented by the SiteConfig kustomize plugin in the current GitOps Zero Touch Provisioning (ZTP) flow.
+The SiteConfig Operator introduces a unified ClusterInstance API, which is derived from the SiteConfig API from the SiteConfig Generator kustomize plugin. This API decouples parameters that define a cluster from the manner in which the cluster is deployed. This separation removed certain limitations that were presented by the SiteConfig kustomize plugin in the past.
 
-Because of this decoupling, the SiteConfig Operator allow deploying OpenShift clusters using either the {ai-workflow}[Agent Based Installation] or {ibi-docs}[Image Based Installation] flows depending on the user's needs. See as well, that the operator is the entry point for any GitOps ZTP deployment.
+Because of this decoupling, the SiteConfig Operator allows deploying OpenShift clusters using either the {ai-workflow}[Agent Based Installation] or {ibi-docs}[Image Based Installation] flows depending on the user's needs. See as well, that the operator is the entry point for any GitOps ZTP deployment.
```

Comment:
I think this can be removed since the siteconfig operator is not used in IBI.

Reply:
Well it's used to render the required templates, same as with Agent Based Installation. I'd keep it.